sql/moz-fx-data-shared-prod/accounts_db_external/fxa_recovery_phones_v1/schema.yaml

```diff
@@ -0,0 +1,17 @@
+fields:
+- name: uid
+  type: INTEGER
+  mode: NULLABLE
+  description: Account ID in hexadecimal format.
+- name: phoneNumber
+  type: STRING
+  mode: NULLABLE
+- name: createdAt
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: lastConfirmed
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: lookupData
+  type: JSON
```

Comment:
Is `type: JSON` legit?

Reply:
Yes, it's [legit](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type).